ironic/drivers/modules/drac/management.py

```diff
@@ -183,10 +189,17 @@ def set_boot_device(node, device, persistent=False):
     :raises: DracOperationError on an error from python-dracclient.
     """
 
-    drac_job.validate_job_queue(node)
-
     client = drac_common.get_drac_client(node)
 
+    unfinished_jobs = drac_job.list_unfinished_jobs(node)
+    if unfinished_jobs:
+        bios_jobs = \
+            [job for job in unfinished_jobs if _BIOS_JOB_NAME in job.name]
+        if bios_jobs:
+            client.delete_jobs(job_ids=[_CLEAR_JOB_IDS])
+            node.driver_internal_info['clear_job_substep'] = 'completed'
+            node.save()
```

Comment:
if we do the above, then i don't think we'll need the above 2 lines or the periodic job and associated method either.

Reply:
I have removed this 2 lines.